paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetFlowType.kt

```diff
@@ -0,0 +1,6 @@
+package com.stripe.android.paymentsheet.ui
+
+internal enum class PaymentSheetFlowType {
```

Comment:
Unfortunately necessary right now for the primary button. We can probably remove the need for it by moving to the composable `PrimaryButton` soon.